x/gov/client/rest/rest.go

```diff
@@ -429,7 +465,27 @@ func queryVotesOnProposalHandlerFn(cdc *codec.Codec, cliCtx context.CLIContext)
 			return
 		}
 
-		res, err := cliCtx.QueryWithData("custom/gov/votes", bz)
+		res, err := cliCtx.QueryWithData("custom/gov/proposal", bz)
+		if err != nil {
+			utils.WriteErrorResponse(w, http.StatusInternalServerError, err.Error())
+			return
+		}
+
+		var proposal gov.Proposal
+		if err := cdc.UnmarshalJSON(res, &proposal); err != nil {
+			utils.WriteErrorResponse(w, http.StatusInternalServerError, err.Error())
+			return
+		}
+
+		// For inactive proposals we must query the txs directly to get the votes
+		// as they're no longer in state.
+		propStatus := proposal.GetStatus()
+		if !(propStatus == gov.StatusVotingPeriod || propStatus == gov.StatusDepositPeriod) {
```

Comment:
Ditto. This should only be !votingperiod

Reply:
See above comment.

Reply:
see response to comment